server/spec/autobind_disabled_for_owner_spec.rb

```diff
@@ -9,6 +9,7 @@
   before(:each) do
     owner = create_owner random_string('test_owner1')
     owner['autobindDisabled'] = true
+    owner['autobindHyperVisorDisabled'] = true
```

Comment:
Hypervisor is one word; the V should be lowercase here, and in all other instances as well.

Reply:
Ok will make that change